### PR TITLE
search bar added

### PR DESCRIPTION
**Pull request by allleeeex**

### CODE REVIEW

**Review by JinTheHuman — APPROVED** [review]
Nice

File: frontend/src/pages/ExplorePage.tsx [hunk 1]
```diff
@@ -202,21 +217,72 @@ const ExplorePage = () => {
...
+      "gender": "Female", 
+      "floor": "Floor 1",
+      "favourited": "true",
+    },
+  ];
+
+  let toiletDisplay = [...toiletCollection];
+
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    var lowerCase = event.target.value.toLowerCase();
```
Comment (oliverxu42) [inline frontend/src/pages/ExplorePage.tsx]: No `var` please

File: frontend/src/pages/ExplorePage.tsx [hunk 2]
```diff
@@ -375,11 +401,22 @@ const ExplorePage = () => {
...
+            label="Search"
+            style={{ backgroundColor: 'white', borderRadius: '5px' }}
+            />
+          </SearchBar>
 
-          <div style={{ fontSize: '18px', fontWeight:200 }}>Finding your Perfect Shit</div>
+          <div style={{ fontSize: '20px', fontWeight:200, paddingLeft: '50px' }}>Finding your Perfect Shit</div>
 
-          <IconButton style={{ padding: '20px' }} onClick={languageTrue}>
+          <IconButton style={{ padding: 0 }} onClick={languageTrue}>
```
Comment (oliverxu42) [inline frontend/src/pages/ExplorePage.tsx]: No in-line css pls change to a styled component.

File: frontend/src/pages/ExplorePage.tsx [hunk 3]
```diff
@@ -202,21 +217,72 @@ const ExplorePage = () => {
   const [languageOpen, setLanguageOpen] = useState(false);
   const [openProfile, setOpenProfile] = useState(false);
   const [isFilterOpen, setIsFilterOpen] = useState(false);
+  const [favourite, setFavourite] = useState("");
   const [gender, setGender] = useState<string>("");
-  const [locations, setLocations] = useState<{ [key: string]: boolean }>({});
+  const [searchInput, setSearchInput] = useState("");
+
+  const toiletCollection = [
```
Comment (oliverxu42) [inline frontend/src/pages/ExplorePage.tsx]: Can you chuck these into a json file and import them so they can be used in other components?

**Review by oliverxu42 — CHANGES_REQUESTED** [review]

**Review by NE1NN — APPROVED** [review]
looks good

**Review by oliverxu42 — APPROVED** [review]

**Pull request merged**